docs/01-overview/02-main-areas/observability/obsv-10-useful-links.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Observability - Useful links
+---
+
+If you're interested in learning more about the Observability area, check out these links:
+
+- Learn how to set up the [Monitoring Flow](../../../03-tutorials/observability/obsv-01-monitoring-overview.md) for your services in Kyma, in which you [observe your application metrics](../../../03-tutorials/observability/obsv-02-observe-application-metrics.md), [set up your Grafana Dashboard](../../../03-tutorials/observability/obsv-03-create-and-configure-grafana-dashboard.md), [define alerting rules](../../../03-tutorials/observability/obsv-04-define-alerting-rules-monitor.md), and eventually, [send notifications](../../../03-tutorials/observability/obsv-05-send-notifications.md). ...And remember to [clean up when you're done](../../../03-tutorials/observability/obsv-06-clean-up-configuration.md) ;)
```

Comment:
```suggestion
- Learn how to set up the [Monitoring Flow](../../../03-tutorials/observability/obsv-01-monitoring-overview.md) for your services in Kyma, in which you [observe your application metrics](../../../03-tutorials/observability/obsv-02-observe-application-metrics.md), [set up your Grafana Dashboard](../../../03-tutorials/observability/obsv-03-create-and-configure-grafana-dashboard.md), [define alerting rules](../../../03-tutorials/observability/obsv-04-define-alerting-rules-monitor.md), and eventually, [send notifications](../../../03-tutorials/observability/obsv-05-send-notifications.md). And remember to [clean up when you're done](../../../03-tutorials/observability/obsv-06-clean-up-configuration.md). 
```

1. The punctuation `. ...` seemed a bit strange to me. Maybe it would be better without the `...` or if the last sentence was on a new line? 
2. I love the more informal tone, but because our current docs are more formal - an emoji looks out of place 😞 But feel free to leave it. (Maybe the tone of the documentation is another thing we can change in Kyma 2.0?)